ci/pipelines/deploy-to-test.yml

```diff
@@ -4117,26 +4123,31 @@ jobs:
           image: telegraf-ecr-registry-test/image.tar
           additional_tags: telegraf-ecr-registry-test/tag
 
-  - name: push-notifications-to-test-ecr
+  - name: build-and-push-notifications-to-test-ecr
     plan:
       - get: pay-ci
       - get: notifications-git-release
         trigger: true
-      # Temporarily fetch image from Dockerhub until Concourse can build its own
-      - <<: *pull-image-from-dockerhub
-        input_mapping:
-          git-release: notifications-git-release
+      - task: build-notifications-image
+        privileged: true
         params:
-          DOCKER_REPOSITORY: govukpay/notifications
-          <<: *docker_credentials
-      - task: parse-release-tag
-        file: pay-ci/ci/tasks/parse-release-tag.yml
-        input_mapping:
-          git-release: notifications-git-release
+            CONTEXT: notifications-git-release
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: concourse/oci-build-task
+          inputs:
+            - name: notifications-git-release
+          outputs:
+            - name: image
+          run:
+            path: build
       - put: notifications-ecr-registry-test
         params:
           image: image/image.tar
-          additional_tags: tags/tags
+          additional_tags: notifications-git-release/.git/ref
```

Comment:
This isn't the right tag, this is pushing with the `alpha_release-XX` tag instead of XXX-release

You want to use the parse-release-tag task still in order to parse to get a XXX-release tag

Reply:
[Fixed](https://github.com/alphagov/pay-ci/pull/644/commits/54b3d402be1839fccc44e6902f11b05e1d8c1ed3)

Reply:
https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/deploy-to-test/jobs/build-and-push-notifications-to-test-ecr/builds/3